### PR TITLE
Show max sold or min received in swap details content

### PR DESCRIPTION
**Pull request by jinchung**
PoW: https://recordit.co/JsFm9g3URZ

### CODE REVIEW

File: src/hooks/useSwapAdjustedAmounts.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import {
+  computeSlippageAdjustedAmounts,
+  Field,
+} from '@rainbow-me/handlers/uniswap';
+import { AppState } from '@rainbow-me/redux/store';
+import { SwapModalField } from '@rainbow-me/redux/swap';
+import { updatePrecisionToDisplay } from '@rainbow-me/utilities';
+
+export default function useSwapAdjustedAmounts(tradeDetails: Trade) {
+  const genericAssets = useSelector(
```
Comment (osdnk) [inline src/hooks/useSwapAdjustedAmounts.ts:12]: isn't this code copy-pasted?

File: src/redux/swap.ts [hunk 2]
```diff
@@ -186,6 +197,7 @@ const INITIAL_STATE: SwapState = {
   inputCurrency: null,
   isMax: false,
   outputCurrency: null,
+  slippageInBips: 50,
```
Comment (osdnk) [inline src/redux/swap.ts:200]: Didn't we get rid of this word?

**Review by osdnk — APPROVED** [review]
can you add some screenshots?

File: src/redux/swap.ts (hunk 2 above)
Comment (jinchung) [reply src/redux/swap.ts:200]: This is new! We used to mix up slippage with price impact. Now, price impact is separate.
This slippageInBips is to prepare for when we allow users to explicitly update the slippage they are comfortable with for a trade. It is not fully integrated yet, but I need this in the redux state for future use.

File: src/hooks/useSwapAdjustedAmounts.ts (hunk 1 above)
Comment (jinchung) [reply src/hooks/useSwapAdjustedAmounts.ts:12]: I'm not sure what you mean - I'm reusing the computeSlippageAdjustedAmounts but we have not displayed the minimum amount received/maximum amount sold anywhere.

**Comment by jinchung** [comment]
> can you add some screenshots?

I added a recording in the description

**Pull request merged**